### ISSUE TITLE
In R-CNN evaluation, key 'binary_cls' does not exist

### ISSUE THREAD

**Issue by MatthewInkawhich** [deeplearning-wisc/siren#5]
Hello. I'm trying to replicate the R-CNN results on VOC/COCO but I'm running into an issue with the voc_coco_vmf.py and voc_coco_knn.py metric computations. 

The line:
`id_val_data = torch.stack(id_val_data['binary_cls']).cpu()`

Errors because there is no `binary_cls` key stored in the probabilistic_scoring_res_odd_{}.pkl files.

How is this computed?
Thanks.

**Comment by d12306** [comment]
hi, @MatthewInkawhich , thanks for the info! It is indeed a bug, I have fixed it.

**Comment by d12306** [comment]
please retry and let me know if there are any further issues on that.
